MODELS.md

```diff
@@ -0,0 +1,122 @@
+
+Based on [Parikh et al, 2017](https://www.semanticscholar.org/paper/A-Decomposable-Attention-Model-for-Natural-Languag-Parikh-T%C3%A4ckstr%C3%B6m/07a9478e87a8304fc3267fa16e83e9f3bbd98b27)
+
+```
+$ docker run allennlp/allennlp:v0.6.1 \
+    evaluate \
+    https://s3-us-west-2.amazonaws.com/allennlp/models/decomposable-attention-elmo-2018.02.19.tar.gz \
+    https://s3-us-west-2.amazonaws.com/allennlp/datasets/snli/snli_1.0_test.jsonl
+  
+Metrics:
+accuracy: 0.864
+```
+
+## Semantic Role Labeling
+
+### [srl-model-2018.05.25.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/srl-model-2018.05.25.tar.gz) (697 MB)
+
+Based on [He et al, 2017](https://www.semanticscholar.org/paper/Deep-Semantic-Role-Labeling-What-Works-and-What-s-He-Lee/a3ccff7ad63c2805078b34b8514fa9eab80d38e9)
+
+```
+f1: 0.849
+```
+
+
+## Coreference Resolution
+
+### [coref-model-2018.02.05.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/coref-model-2018.02.05.tar.gz) (56 MB)
+
+Based on [End-to-End Coreference Resolution (Lee et al, 2017)](https://www.semanticscholar.org/paper/End-to-end-Neural-Coreference-Resolution-Lee-He/3f2114893dc44eacac951f148fbff142ca200e83)
+
+```
+f1: 0.630
+```
+
+## Named Entity Recognition
+
+### [ner-model-2018.04.26.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.04.26.tar.gz) (678.9 MB)
+
+Based on [Deep contextualized word representations](https://arxiv.org/abs/1802.05365)
+
+```
+f1: 0.925
+```
+
+### [fine-grained-ner-model-elmo-2018.08.31.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/fine-grained-ner-model-elmo-2018.08.31.tar.gz) (691 MB)
+
+
+## Constituency Parsing
+
+### [elmo-constituency-parser-2018.03.14.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/elmo-constituency-parser-2018.03.14.tar.gz) (678 MB)
+
+Based on [Minimal Span Based Constituency Parser (Stern et al, 2017)](https://www.semanticscholar.org/paper/A-Minimal-Span-Based-Neural-Constituency-Parser-Stern-Andreas/593e4e749bd2dbcaf8dc25298d830b41d435e435) but with ELMo embeddings
+
+
+## Dependency Parsing
+
+### Biaffine Parser
+
+Based on [Dozat and Manning, 2017](https://arxiv.org/pdf/1611.01734.pdf)
+
+* [biaffine-dependency-parser-ptb-2018.08.23.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/biaffine-dependency-parser-ptb-2018.08.23.tar.gz) (69 MB) uses [Penn Treebank](https://catalog.ldc.upenn.edu/ldc99t42) style dependencies.
+
+* [biaffine-dependency-parser-ud-2018.08.23.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/biaffine-dependency-parser-ud-2018.08.23.tar.gz) (61 MB) uses [Universal Dependency](http://universaldependencies.org/) style depedencies.
+
+```
+f1: 0.941
+```
+
+## Semantic Parsing
+
+### Wikitables
+
+#### [wikitables-model-2018.09.14.tar.gz](https://s3-us-west-2.amazonaws.com/allennlp/models/wikitables-model-2018.09.14.tar.gz)
```

Comment:
@matt-gardner what is the preliminary performance of this model? Is it worth writing down?

Reply:
I don't think it's worth writing down.  I'd instead just put major caveats that this is trained on only part of the data and not officially evaluated.  It'd just a placeholder.